app/utils/Config.scala

```diff
@@ -0,0 +1,43 @@
+package utils
+
+import org.yaml.snakeyaml.Yaml
+import scala.io.Source
+import scala.collection.JavaConversions._
+import scala.collection.mutable.Map
+import java.util.LinkedHashMap
+
+class Config(config_fn: String) {
+  private var api_key = ""
```

Comment:
Also, can you make this name more specific? We can use this `Config` class for accessing all of our API keys (Twilio has some too).